src/condor_tools/htcondor_cli/annex_create.py

```diff
@@ -668,53 +657,34 @@ def annex_create(
                 else:
                     schedd.edit(job_id, "TransferInput", "undefined")
 
-
-# FIXME: If we really care about this functionality, use argparse and duplicate
-# the args from the htcondor cli tool
-def __main():
-    """
-    Main entry point, only to be used when run as a standalone executable
-    """
-    username = getpass.getuser()
-    target = "stampede2"
-    nodes = 2
-    lifetime = 7200
-
-    annex_name = "hpc-annex"
-    queue_name = "development"
-    owners = username
-    allocation = None
-
-    collector = htcondor.param.get(
-        "ANNEX_COLLECTOR", "htcondor-cm-hpcannex.osgdev.chtc.io"
-    )
-    token_file = f"~/.condor/tokens.d/{username}@annex.osgdev.chtc.io"
-    control_path = "~/.hpc-annex"
-    password_file = htcondor.param.get(
-        "ANNEX_PASSWORD_FILE", "~/.condor/annex_password_file"
-    )
-
-    ssh_target = f"{username}@login.xsede.org"
-
-    logging.basicConfig(format="%(message)s", level=logging.INFO)
-    logger = logging.getLogger()
-
-    annex_create(
-        logger,
-        annex_name,
-        nodes,
-        lifetime,
-        allocation,
+    remotes = {}
+    logger.info(f"Submitting SLURM job on {target}:\n")
+    rc = invoke_pilot_script(
+        ssh_connection_sharing,
+        ssh_target,
+        ssh_indirect_command,
+        remote_script_dir,
         target,
+        annex_name,
         queue_name,
-        owners,
         collector,
         token_file,
+        lifetime,
+        owners,
+        nodes,
+        allocation,
+        lambda attribute, value: updateJobAd(cluster_id, attribute, value, remotes),
+        request_id,
         password_file,
-        ssh_target,
-        control_path,
     )
 
+    if rc == 0:
+        logger.info(f"... remote SLURM job submitted.")
+    else:
+        error = f"Failed to start annex, SLURM returned code {rc}"
+        try:
+            schedd.act(htcondor.JobAction.Remove, f'ClusterID == {cluster_id}', error)
+        except:
```

Comment:
```suggestion
        except Exception:
```

Never use bare `except:` (see e.g. https://stackoverflow.com/a/54948581 )

Reply:
And I'm noticing now that I also failed to fix this earlier in `annex_create.py`:

```python
    except:
        raise RuntimeError(f"Failed to submit state-tracking job, aborting.")
```

Reply:
Done.